<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Connection and start</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ns</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>inRate:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>count/s</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_peakingTime"/>
       </item>
       <item row="3" column="1" rowspan="2">
        <widget class="QComboBox" name="comboBox_numOfBins"/>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>liveTime:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QLineEdit" name="lineEdit_inCount"/>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_bytePerBin"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>measureTime：</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEdit_outRate"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>numOfBins:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_outCount"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_realTime"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_measureTime"/>
       </item>
       <item row="8" column="4">
        <widget class="QLineEdit" name="lineEdit_inRate"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>outRate:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>peakingTime：</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>inCount:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="5">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>count</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>s</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>realTime:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>count</string>
         </property>
        </widget>
       </item>
       <item row="6" column="5">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>s</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>bytePerBin:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="5">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>count/s</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="pushButton_setPeakingTime">
         <property name="text">
          <string>set</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QLineEdit" name="lineEdit_liveTime"/>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="pushButton_setMeasureTime">
         <property name="text">
          <string>set</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4" rowspan="2">
        <widget class="QPushButton" name="pushButton_startRun">
         <property name="text">
          <string>startRun</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>s</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="6">
        <widget class="QTableWidget" name="tableWidget_device">
         <property name="rowCount">
          <number>1</number>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <row>
          <property name="text">
           <string/>
          </property>
         </row>
         <column>
          <property name="text">
           <string>serialNumber</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>interface</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>remark</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>outCount:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_1">
      <property name="title">
       <string>Information Output</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit_infoOutput"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_VICODaemon">
    <property name="title">
     <string>VICODaemon Service</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionclose"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QMenu" name="menuscan">
     <property name="title">
      <string>scan</string>
     </property>
     <addaction name="actionUSB"/>
     <addaction name="actionEthernet"/>
    </widget>
    <addaction name="menuscan"/>
    <addaction name="actionrefresh"/>
   </widget>
   <addaction name="menu_VICODaemon"/>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionUSB">
   <property name="text">
    <string>USB</string>
   </property>
  </action>
  <action name="actionEthernet">
   <property name="text">
    <string>Ethernet</string>
   </property>
  </action>
  <action name="actionrefresh">
   <property name="text">
    <string>refresh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
